depends/README.md

```diff
@@ -12,15 +12,18 @@ For example:
 
     make HOST=x86_64-w64-mingw32 -j4
 
-**Bitcoin Core's configure script by default will ignore the depends output.** In
+**Bitcoin Core's `configure` script by default will ignore the depends output.** In
 order for it to pick up libraries, tools, and settings from the depends build,
-you must point it at the appropriate `--prefix` directory generated by the
-build. In the above example, a prefix dir named x86_64-w64-mingw32 will be
-created. To use it for Bitcoin:
+you must set the `CONFIG_SITE` environment variable to point to a `config.site` settings file.
+In the above example, a file named `depends/x86_64-w64-mingw32/share/config.site` will be
+created. To use it during compilation:
 
-    ./configure --prefix=$PWD/depends/x86_64-w64-mingw32
+    CONFIG_SITE=$PWD/depends/x86_64-w64-mingw32/share/config.site ./configure
 
-Common `host-platform-triplets` for cross compilation are:
+The default install prefix when using `config.site` is `--prefix=depends/<host-platform-triplet>`,
+so depends build outputs will be installed in that location.
```

Comment:
By "depends build outputs" I'm assuming you mean things like the fontconfig tools, which will end up installed into `depends/<host-platform-triplet>`. i.e:
```bash
# ls depends/x86_64-pc-linux-gnu/bin/
event_rpcgen.py  fc-cache  fc-cat  fc-list  fc-match  fc-pattern  fc-query  fc-scan  fc-validate  freetype-config  xmlwf
```

Is that correct?

Reply:
All of the dependency packages are installed into `depends/<host-platform-triplet>`, using `bin`, `lib` and other subdirectories.

The `make -C depends` rightfully reports:
```
copying packages: native_b2 boost libevent qt expat libxcb xcb_proto libXau xproto freetype fontconfig libxkbcommon qrencode bdb sqlite miniupnpc libnatpmp zeromq
to: /home/hebasto/GitHub/bitcoin/depends/x86_64-pc-linux-gnu
```

---

Not to be confused with the default `--prefix` for Bitcoin Core, that is `/usr/local`. That means `make install` will install `bitcoind` into `/usr/local/bin`.